src/main/java/com/basho/riak/client/core/converters/TimeSeriesPBConverter.java

```diff
@@ -73,7 +85,7 @@ public static QueryResult convertPbGetResp(RiakKvPB.TsQueryResp response)
         return pbRows;
     }
 
-    private static ArrayList<RiakKvPB.TsCell> convertCellsToPb(Collection<Cell> cells)
+    public static ArrayList<RiakKvPB.TsCell> convertCellsToPb(Collection<Cell> cells)
```

Comment:
it is better to return more generic List rather than return a concrete ArrayList